subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf5_ppi.h

```diff
@@ -305,33 +315,21 @@
 	((u32_t)&(NRF_RADIO->TASKS_RXEN))
 
 #if defined(CONFIG_SOC_NRF52840)
-/* The 2 adjacent TIMER EVENTS_COMPARE event offsets used for implementing
```

Comment:
These changes and the ones below are due to the fact that we reserve a single PPI for S2.